src/pages/verifications/createUpdateVerification/useCreateUpdateVerification.tsx

```diff
@@ -113,6 +158,34 @@ export const useCreateUpdateVerification = () => {
   const name = watch('name');
   const description = watch('description');
 
+  const onChangeAttribute = (index: number, attribute?: OptionType, operator?: OptionType, value?: string) => {
+    const newAttributes = getValues().attributes;
+    if (attribute !== undefined) {
```

Comment:
specifically `undefined` ? if attribute is null, what happens?

Reply:
This function doesn't accept null for attribute. it is either undefined or has value

Reply:
i mean you can easilyc check with `(attribute)` as truthy value